extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -51,29 +59,30 @@ export class BuildHelper {
 			await fs.mkdir(this.extensionBuildDir);
 		}
 
-		const buildfilesPath = await this.getBuildDirPathFromMssqlTools();
-		buildFiles.forEach(async (fileName) => {
+		// download the Microsoft.Build.Sql sdk nuget
+		const httpClient = new HttpClient();
```

Comment:
do you think the files should be redownloaded if they already exist?  it looks like they are only be downloaded once per-process lifetime but if ADS is restarted they'd be redownloaded an extra time, if I'm reading correctly?

Reply:
yeah I was initially just having this have the same behavior as it currently is, where the files are always copied over from STS with the first build, but makes sense to only download if there's a new version.

Reply:
updated this to no longer redownload on each ADS restart, only if the nuget version does not match the expected one. The nuget no longer gets deleted after extracting